trackers_integration/issuetracker/openproject.py

```diff
@@ -234,7 +234,12 @@ def details(self, url):
         issue = self.rpc.get_workpackage(self.bug_id_from_url(url))
         issue_type = issue["_embedded"]["type"]["name"].upper()
         status = issue["_embedded"]["status"]["name"].upper()
+        try:
+            assignee = issue["_embedded"]["assignee"]["name"]
+        except Exception as err:
+            assignee = "unassigned"
+
         return {
             "title": f"{status} {issue_type}: " + issue["subject"],
-            "description": issue["description"]["html"],
+            "description": "Assignee: " + assignee,
```

Comment:
This is a change in behavior so it's not acceptable as-is!

1) The actual issue with the description needs to be fixed
2) To show the assignee you can either
  2.1) Append or prepend that to the description or;
  2.2) inherit from the current class and provide a custom implementation for your own instance.